typings/wavesurfer-defs.d.ts

```diff
@@ -0,0 +1,73 @@
+// Code - https://raw.githubusercontent.com/katspaugh/wavesurfer.js/master/src/wavesurfer.js
```

Comment:
Wanted to check whether we need to update this whenever we update the wave surfer version?

Reply:
Yup, will add a lint check later on for this too.

Reply:
> Yup, will add a lint check later on for this too.

Do we have any (github) issue for that? Also, how would we catch this till we have this check?

Reply:
Ah, actually the lint check will not check that the code link is correct. It will be there so that the dev is aware that the type defs were for a previous version and they may have changed now.

Reply:
Still, my question persists i.e, until we don't have that "warning through lint checks" how would we catch this during any update?

Reply:
I guess we can do that manually, as I don't think type defs would be updated so frequently.

Reply:
@DubeySandeep Since I'm mostly responsible for library upgrades I can make sure that the type definitions are correct until we have the lint check.